src/bootstrap.php

```diff
@@ -12,10 +12,9 @@
 use Go\ParserReflection\ReflectionEngine;
 
 /**
- * This file is used for automatic configuration of Go\ParserReflection\ReflectionEngine class
+ * This file is used for automatic configuration of
+ * Go\ParserReflection\ReflectionEngine class
  */
 ReflectionEngine::init(new ComposerLocator());
-// Polifyll for PHP<7.0
-if (!class_exists(ReflectionType::class, false)) {
-    class ReflectionType {}
-}
+
+require(__DIR__ . '/polyfill.php');
```

Comment:
Hmm... I think this is a case of minor improvement in PSR-1 compliance for 1 extra small file load once, during execution. I think it's a worthwhile cost... and I think having Polyfills in a file called polyfill.php helps people know where to find them.